### PR TITLE
Added Letheia Nisei

### PR DESCRIPTION
**Pull request by x3N1GM4x**
As per http://nisei.net/article/2019AScoop7.

Please find my mistakes! 😛

### CODE REVIEW

**Comment by x3N1GM4x** [comment]
Ruh roh, wonder what I messed up, lol. 😄

**Comment by gereons** [comment]
LGTM!

**Comment by x3N1GM4x** [comment]
> LGTM!

Travis disagrees, lol. I can see that it's valid json in general terms, so I guess I've contravened the schema somewhere, although I can't spot where yet...

**Comment by gereons** [comment]
Yeah, I wrote that after only looking at the diff. Unsure what the validator is complaining about, but will figure it out.

File: pack/df.json [hunk 1]
```diff
@@ -579,6 +579,24 @@
...
+        "pack_code": "df",
+        "position": 46,
+        "quantity": 3,
+        "side_code": "corp",
+        "text": "The first time each run the runner approaches this server, you and the Runner secretly spend 0[credit], 1[credit] or 2[credit]. Reveal spent credits. If you and the Runner spent a different number of credits, you may trash this upgrade to force the runner to approach the outermost piece of ice protecting this server.",
+        "title": "Letheia Nisei",
+        "trash_cost": 2,
+        "type_code": "upgrade",
+        "uniqueness": true
+    },
```
Comment (gereons) [inline pack/df.json]: the validator is _really_ picky about trailing whitespace. remove the blank spaces after the comma, please.

Comment (x3N1GM4x) [reply pack/df.json]: Aha, good to know, thanks! I'll update and see if I can get it through now.

**Comment by x3N1GM4x** [comment]
Deployed to production - also made the image available. 👍

**Comment by NoahTheDuke** [comment]
You can see what you've messed up locally if you run `python3 validate.py`, and you can even fix it automatically by running `python3 validate.py --fix`!

**Comment by x3N1GM4x** [comment]
> You can see what you've messed up locally if you run `python3 validate.py`, and you can even fix it automatically by running `python3 validate.py --fix`!

Ooooh, thats a top tip, thanks!

**Pull request merged**